src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const separatedByLines = sourceString
+    .split(';')
+    .filter((line) => line.trim() !== '');
+  const stringObj = {};
```

Comment:
The variable name `stringObj` is not descriptive of its purpose. Consider renaming it to something more meaningful, like `styleObject` or `cssProperties`, to enhance code readability.